graphdatascience/procedure_surface/cypher/catalog/node_properties_cypher_endpoints.py

```diff
@@ -0,0 +1,122 @@
+from typing import Any, List, Optional, Union
+
+from pandas import DataFrame
+
+from graphdatascience import Graph, QueryRunner
+from graphdatascience.call_parameters import CallParameters
+from graphdatascience.procedure_surface.api.catalog.node_properties_endpoints import (
+    NodePropertiesDropResult,
+    NodePropertiesEndpoints,
+    NodePropertiesWriteResult,
+    NodePropertySpec,
+)
+from graphdatascience.procedure_surface.utils.config_converter import ConfigConverter
+from graphdatascience.procedure_surface.utils.result_utils import transpose_property_columns
+from graphdatascience.query_runner.gds_arrow_client import GdsArrowClient
+
+
+class NodePropertiesCypherEndpoints(NodePropertiesEndpoints):
```

Comment:
either its more of NodePropertiesPluginEndpoints or i would suggest to split in two.

How about having:

* CypherEndpoints
* ArrowEndpoints
* ? ArrowV1Endpoints 

As we already have the logic to split it per impl.

Might be easier to see once we have the glue code which calls into these endpoint impl.